### PR TITLE
Add monolog support

### PR DESCRIPTION
**Pull request by ameech**
Can't be merged until this PR is merged: https://github.com/equip/framework/pull/4

### CODE REVIEW

**Comment by shadowhand** [comment]
Rebase to master, pretty please.

**Comment by ameech** [comment]
Rebased hard.

File: web/index.php [hunk 1]
```diff
@@ -12,13 +12,15 @@
...
+    Equip\Configuration\MonologConfiguration::class,
 ])
 ->setMiddleware([
     Relay\Middleware\ResponseSender::class,
     Equip\Handler\ExceptionHandler::class,
     Equip\Handler\DispatchHandler::class,
-    Equip\Handler\JsonContentHandler::class,
-    Equip\Handler\FormContentHandler::class,
+    Relay\Middleware\JsonContentHandler::class,
+    Relay\Middleware\FormContentHandler::class,
```
Comment (sctape) [inline web/index.php:23]: Updated to non-deprecated classes

File: composer.json [hunk 2]
```diff
@@ -4,7 +4,9 @@
     "type": "project",
     "license": "MIT",
     "require": {
-        "equip/framework": "^1.0"
+        "equip/framework": "^1.0",
+        "josegonzalez/dotenv": "^2.0",
```
Comment (sctape) [inline composer.json:8]: Figured since the .env file was already committed to the repo, it made sense to include the dotenv library and configuration.

**Comment by shadowhand** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/equip/project/pull-request/2/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=equip/project)

**Pull request merged**